virttest/qemu_vm.py

```diff
@@ -1543,6 +1558,8 @@ def add_disable_modern(devices, dev, dev_type):
                     if self.last_boot_index > 0:
                         image_boot = False
                     self.last_boot_index += 1
+            image_pci_bus = _get_pci_bus(image_params, "hba")
+            image_params["pci_bus"] = image_pci_bus.get("aobject", "pci.0")
```

Comment:
(what I mean is in case of virtio_blk the device is not host->bus adapter, but directly the pci disk device)